### ISSUE TITLE
Forbid extra `object` in parent classes list

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#248]
# Rule request

## Thesis

Today I saw this:

```python
class Some(FirstMixin, SecondMixin, object): ...
```

Which result in the following `mro`:

```python
>>> Some.__mro__
(<class '__main__.Some'>, <class '__main__.FirstMixin'>, <class '__main__.SecondMixin'>, <class 'object'>)
```

## Reasoning

We should allow `object` only when we explicitly use it as a single parent class.
When there is an other class or there are multiple parents - we should not allow it for the consistency reasons.

**Comment by AlexArcPy** [comment]
@sobolevn , started working on this!
